tutorials/scripts/how_to_use_python_scripts.md

```diff
@@ -0,0 +1,137 @@
+# Python Scripts
+
+## Installing Python 3.6
+
+### Windows
+
+Download the latest version of Python 3.6 from [here](https://www.python.org/downloads/), then run the installer. We will guide you through the installation.
+
+![pic1](images/python36_windows_1.png)
+
+Make sure the option "Add Python 3.6 to PATH" is checked. Otherwise you won't be able to run scripts from anywhere in the filesystem. Choose "Customize installation" for the next step.
+
+![pic2](images/python36_windows_2.png)
+
+The option "pip" has to be checked. All other options are not mandatory, but can be helpful when working with Python. Click "Next" to continue.
+
+![pic3](images/python36_windows_3.png)
+
+The option "Add Python to environment variables" has to be checked. This is required to get access to the Python package management `pip`. Click "Install".
+
+To check if everything was installed correctly type `python --version` into *Powershell*. It should display the Python 3.6 version.
+
+### Linux
+
+**Ubuntu/Debian**
+
+Open you terminal and type
+
+```
+sudo apt install python3 python3-pip
```

Comment:
we need pillow, right? so we can install pillow directly without pip (much better: auto-updates, ...)

Reply:
I intended this to be a general guide about how to get python, not specifically for the terrain script. The blender scripts don't use pillow for example.

Reply:
ah, i see.